marketplace/commercetools-products/extension.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "commercetools-products",
+  "name": "commercetools Products",
+  "srcdoc": "./build/index.html",
+  "fieldTypes": [
+    "Symbol", "Symbols"
+  ],
+  "parameters": {
+    "installation": [
+      {
+        "id": "projectKey",
+        "name": "commercetools Project Key",
+        "type": "Symbol",
+        "required": true
+      },
+      {
+        "id": "clientId",
+        "name": "commercetools Authentication Client ID",
+        "type": "Symbol",
+        "required": true
+      },
+      {
+        "id": "clientSecret",
+        "name": "commercetools Authentication Client Secret",
```

Comment:
Is it safe to have this value exposed in the frontend?

Reply:
Yes, it can be scoped to only allow reading products.